{"uid":"4b4757e66a1912dae1a509f688f20b0f","name":"categories","children":[{"name":"Product defects","children":[{"name":"snappi_ixnetwork.exceptions.SnappiIxnException:   File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/snappi_api.py\", line 430, in set_control_state\n    self.ngpf.set_protocol_state(request_payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/device/ngpf.py\", line 256, in set_protocol_state\n    self.api._ixnetwork.StartAllProtocols(\"sync\")\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/testplatform/sessions/ixnetwork/ixnetwork.py\", line 2862, in StartAllProtocols\n    return self._execute(\"startAllProtocols\", payload=payload, response_object=None)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/base.py\", line 452, in _execute\n    response = self._connection._execute(url, payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 289, in _execute\n    return self._send_recv(\"POST\", url, payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 572, in _send_recv\n    return self._poll()\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 640, in _poll\n    return self._process_response_status_code(\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 476, in _process_response_status_code\n    raise BadRequestError(message, response.status_code)\n  System Exception Caught. Exception Text: There is no license available for IxN-SMACsec sessions or the feature is not supported. Please check license availability for Perpetual Mode or check the Release Notes for supported features.\nThere is no license available for IxN-SMACsec sessions or the feature is not supported. Please check license availability for Perpetual Mode or check the Release Notes for supported features.\n\nSystem Exception Caught. Exception Text: There is no license available for IxN-SMACsec sessions or the feature is not supported. Please check license availability for Perpetual Mode or check the Release Notes for supported features.\nThere is no license available for IxN-SMACsec sessions or the feature is not supported. Please check license availability for Perpetual Mode or check the Release Notes for supported features.","children":[{"name":"test_mka_stats","uid":"4e924449c5458b2e","parentUid":"21c190d546db6d4efddb83a087b507c5","status":"failed","time":{"start":1741375664779,"stop":1741375732572,"duration":67793},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"21c190d546db6d4efddb83a087b507c5"},{"name":"snappi_ixnetwork.exceptions.SnappiIxnException:   File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/snappi_api.py\", line 430, in set_control_state\n    self.ngpf.set_protocol_state(request_payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/device/ngpf.py\", line 256, in set_protocol_state\n    self.api._ixnetwork.StartAllProtocols(\"sync\")\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/testplatform/sessions/ixnetwork/ixnetwork.py\", line 2862, in StartAllProtocols\n    return self._execute(\"startAllProtocols\", payload=payload, response_object=None)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/base.py\", line 452, in _execute\n    response = self._connection._execute(url, payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 289, in _execute\n    return self._send_recv(\"POST\", url, payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 572, in _send_recv\n    return self._poll()\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 640, in _poll\n    return self._process_response_status_code(\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 476, in _process_response_status_code\n    raise BadRequestError(message, response.status_code)\n  System Exception Caught. Exception Text: There is no license available for IxN-SMACsec sessions or the feature is not supported. Please check license availability for Perpetual Mode or check the Release Notes for supported features.\nThere is no license available for IxN-SMACsec sessions or the feature is not supported. Please check license availability for Perpetual Mode or check the Release Notes for supported features.\nThere is no license available for IxN-SMACsec sessions or the feature is not supported. Please check license availability for Perpetual Mode or check the Release Notes for supported features.\nThere is no license available for IxN-SMACsec sessions or the feature is not supported. Please check license availability for Perpetual Mode or check the Release Notes for supported features.\n\nSystem Exception Caught. Exception Text: There is no license available for IxN-SMACsec sessions or the feature is not supported. Please check license availability for Perpetual Mode or check the Release Notes for supported features.\nThere is no license available for IxN-SMACsec sessions or the feature is not supported. Please check license availability for Perpetual Mode or check the Release Notes for supported features.\nThere is no license available for IxN-SMACsec sessions or the feature is not supported. Please check license availability for Perpetual Mode or check the Release Notes for supported features.\nThere is no license available for IxN-SMACsec sessions or the feature is not supported. Please check license availability for Perpetual Mode or check the Release Notes for supported features.","children":[{"name":"test_encrypt_with_mka","uid":"25a35554f09ede1c","parentUid":"5d1e3240244571dfac6c52e5fb3b0ab6","status":"failed","time":{"start":1741375664779,"stop":1741375726992,"duration":62213},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"5d1e3240244571dfac6c52e5fb3b0ab6"},{"name":"snappi_ixnetwork.exceptions.SnappiIxnException: bad request errors from Ixn:incrementingPn","children":[{"name":"test_encrypt","uid":"99d0e4a554728bc5","parentUid":"012d03a87864877ef2cfaf3a54c59b24","status":"failed","time":{"start":1741375664779,"stop":1741375711557,"duration":46778},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"012d03a87864877ef2cfaf3a54c59b24"},{"name":"snappi_ixnetwork.exceptions.SnappiIxnException:   File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/snappi_api.py\", line 438, in set_control_state\n    self.traffic_item.transmit(request_payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/trafficitem.py\", line 1304, in transmit\n    self._api._traffic.Apply()\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/testplatform/sessions/ixnetwork/traffic/traffic.py\", line 1395, in Apply\n    return self._execute(\"apply\", payload=payload, response_object=None)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/base.py\", line 452, in _execute\n    response = self._connection._execute(url, payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 289, in _execute\n    return self._send_recv(\"POST\", url, payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 572, in _send_recv\n    return self._poll()\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 640, in _poll\n    return self._process_response_status_code(\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 476, in _process_response_status_code\n    raise BadRequestError(message, response.status_code)\n  Error in L2/L3 Traffic Apply","children":[{"name":"test_vxlan_b2b","uid":"3a38d74ce60b00e9","parentUid":"d39e9e9bd5b95188d6261f2aa4e9dfaa","status":"failed","time":{"start":1741375664779,"stop":1741375829147,"duration":164368},"flaky":true,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"d39e9e9bd5b95188d6261f2aa4e9dfaa"},{"name":"assert 0 == 1\n +  where 0 = len(<ixnetwork_restpy.testplatform.sessions.ixnetwork.availablehardware.chassis.chassis.Chassis object at 0x7f127e8743c0>)","children":[{"name":"test_counter_ip_dscp","uid":"f7e1e6c31a7e56d3","parentUid":"100de879c1d54f8fa1309b4b16e9ef7e","status":"failed","time":{"start":1741375664779,"stop":1741375703514,"duration":38735},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"100de879c1d54f8fa1309b4b16e9ef7e"}],"uid":"8fb3a91ba5aaf9de24cc8a92edc82b5d"}]}